### ISSUE TITLE
Unit tests for event storage

### ISSUE THREAD

**Issue by edwardwilson** [eventflow/EventFlow.MongoDB#9]
Unit tests should be added for event store using the EventFlow.TestHelper

**Comment by edwardwilson** [comment]
@rasmus, @janfeyen I have added a test project into the repo but seem to be having issues running the tests from EventFlow.TestHelpers. Could either of you take a look a look at the project, anything sticking out to you why this wouldn't run?

**Comment by rasmus** [comment]
@edwardwilson There nothing in the build file that executes the test, the task is missing

**Comment by edwardwilson** [comment]
Yes for the appveyor build, but even running with resharper unit test runner or is VS the tests do not run
